### PR TITLE
adding new metrics for replay in storei tile

### PR DESCRIPTION
**Pull request by llamb-jump**

### CODE REVIEW

File: src/app/fdctl/run/tiles/fd_replay.c
Comment (mmcgee-jump) [inline src/app/fdctl/run/tiles/fd_replay.c:519]: I'm not quite sure what these counters are for, but this can't be right. Resetting them to zero will make prometheus think the application exited and restarted, counters should only increment.

Comment (llamb-jump) [reply src/app/fdctl/run/tiles/fd_replay.c]: It made the visualization easier to understand IMO, I removed these lines since they're confusing

**Review by lheeger-jump — APPROVED** [review]

**Review by mmcgee-jump — CHANGES_REQUESTED** [review]
Can we hold off on this one? Needs a discussion before merging, as 1.5 / 2.x probably shouldn't share metrics definitions.

**Comment by llamb-jump** [comment]
how about splitting `metrics.xml` into `metrics-frankendancer.xml` and `metrics-firedancer.xml`?

**Review by mmcgee-jump — APPROVED** [review]

**Pull request merged**
